exercises/leap/cases_test.go

```diff
@@ -1,7 +1,7 @@
 package leap
 
 // Source: exercism/x-common
-// Commit: be6fa53 leap: Rewrite the test cases and their descriptions
+// x-common version: 1.0.0
```

Comment:
Nice, I like having the version, I agree with the advantages of it, but I like it as an addition rather than a replacement, I think the commit pinpoints it to a specific point in time, and includes a brief but useful snippet of info regarding any recent changes.
Though if I had to choose, I'd go for the Version.